### PR TITLE
F 328 write test for donations from wallets

### PR DESCRIPTION
**Pull request by FatemeGhasemi**
Add test for  donationsFromWallets

Related to #328

### CODE REVIEW

**Comment by FatemeGhasemi** [comment]
@mohammadranjbarz can you review this?

File: src/resolvers/donationResolver.ts [hunk 1]
```diff
@@ -167,7 +167,10 @@ export class DonationResolver {
     const fromWalletAddressesArray: string[] = fromWalletAddresses.map(o =>
       o.toLowerCase(),
     );
-
+    logger.debug(
+      'fromWalletAddressesArray after lowercase',
+      fromWalletAddressesArray,
+    );
```
Comment (mohammadranjbarz) [inline src/resolvers/donationResolver.ts]: @FatemeGhasemi  Please remove unused logs

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
@FatemeGhasemi Thanks, everything looks good, and tests passed, so just remove the  unused logs, then we can merge it

**Comment by mohammadranjbarz** [comment]
@FatemeGhasemi Please resolve conflicts

**Review by mohammadranjbarz — APPROVED** [review]
Thanks, LGTM

**Pull request merged**